src/features/layers_in_area/atoms/areaLayersDetailsResource/areaLayersDetailsParamsAtom.ts

```diff
@@ -18,11 +18,9 @@ export const areaLayersDetailsParamsAtom = createAtom(
     state: DetailsRequestParams | null = null,
   ): DetailsRequestParams | null => {
     const layersGlobal = get('layersGlobalResource');
-    const layersInAreaAndEventLayer = get('layersInAreaAndEventLayerResource');
-    const allLayers = [
-      ...(layersGlobal.data ?? []),
-      ...(layersInAreaAndEventLayer.data ?? []),
-    ];
+    const { loading, error, data } = get('layersInAreaAndEventLayerResource');
+    const layersInAreaAndEventLayer = loading || error || !data ? [] : data;
```

Comment:
```suggestion
    const layersInAreaAndEventLayer = (loading || error || !data) ? [] : data;
```

Reply:
we've skipped it due to lint that keeps fixing that